examples/react/kitchen-sink-file-based/src/routes/dashboard.users.tsx

```diff
@@ -68,7 +68,8 @@ function UsersComponent() {
 
   React.useEffect(() => {
     navigate({
-      search: (old) => {
+       to: "/dashboard/users",
```

Comment:
The problem is that a navigate call should be possible without the `to` field.

Reply:
I agree but that is also a problem on main branch already, so it has nothing to do with this fix

Reply:
do you want me to revert the 2 fixes then?